src/components/Controls.vue

```diff
@@ -271,6 +271,8 @@ export default {
 			if (current?.id !== previous?.id) {
 				this.clearFilter()
 			}
+
+			document.title = this.board.title
```

Comment:
Please make sure to store the default page title like "Deck - Nextcloud" that is set and then only add the board title in front. You can find some inspiration on that in the Talk app: https://github.com/nextcloud/spreed/blob/master/src/App.vue#L418-L441

Reply:
Also if no board is set, it should then fall back tio the default one ;)

Reply:
Please check again. All fixed.